packages/sdk/src/react/ui/modals/BuyModal/index.tsx

```diff
@@ -67,6 +69,16 @@ export const BuyModalContent = () => {
 
 	if (modalId == 0 || !collection || !collectable || !buy) return null;
 
+	if (isCheckoutLoading) {
```

Comment:
We have click event listener of cta of `ERC1155QuantityModal`. You can use `buyModal$.isCheckoutLoading.set...`there to have control over loading states. Wdyt?